Makefile

```diff
@@ -0,0 +1,55 @@
+.PHONY: clean requirements
+
+.DEFAULT_GOAL := help
+
+
+help: ### Display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+clean: ## Remove generated byte code, coverage reports, and build artifacts
+	@echo "--> Clean Python files ..."
+	find . -name '__pycache__' -exec rm -rf {} +
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '*~' -exec rm -f {} +
+	rm -fr build/
+	rm -fr dist/
+	rm -fr *.egg-info
+
+	@echo "--> Clean other files ..."
+	rm -fr var/
+	coverage erase
+	@echo "The project has been cleaned successfully."
+
+selfcheck: ## Check that the Makefile is well-formed
+	@echo "The Makefile is well-formed."
+
+test_requirements: ## Install requirements needed by test environment
+	pip install -q -r requirements/quality.txt --exists-action w
+	pip install -q -r requirements/test.txt --exists-action w
+
+requirements: test_requirements ## Installs all requirements needed by developmenent and test environments
+	pip install -q -r requirements/base.txt --exists-action w
+	pip install -e .
+	@echo "Finished installing requirements."
+
+quality:  ## Run quality tests and checks
+	make selfcheck
+	pylint html_xblock tests
```

Comment:
Can you add another line with python 3 compatibility checks and fix any issues? I did it [here on xblock image explorer](https://github.com/edx-solutions/xblock-image-explorer/blob/master/tox.ini#L27).

```
pylint --py3k completable_html_xblock
```

This yields a few errors: 
```
************* Module completable_html_xblock
W:  2, 0: import missing `from __future__ import absolute_import` (no-absolute-import)
************* Module completable_html_xblock.html
W:  2, 0: import missing `from __future__ import absolute_import` (no-absolute-import)
W:  4, 0: import missing `from __future__ import absolute_import` (no-absolute-import)
W:  5, 0: import missing `from __future__ import absolute_import` (no-absolute-import)
W:  6, 0: import missing `from __future__ import absolute_import` (no-absolute-import)
W:  7, 0: import missing `from __future__ import absolute_import` (no-absolute-import)
W:  8, 0: import missing `from __future__ import absolute_import` (no-absolute-import)
W: 12, 0: import missing `from __future__ import absolute_import` (no-absolute-import)
W: 14, 0: import missing `from __future__ import absolute_import` (no-absolute-import)

-----------------------------------
Your code has been rated at 6.67/10
```

Reply:
Good idea!